### ISSUE TITLE
Named circuit breakers

### ISSUE THREAD

**Issue by kpacha** [devopsfaith/krakendesigner#83]
CB accepts a `name` in the config in order to identify itself when logging errors or changes in status

**Comment by alombarte** [comment]
Circuit breaker is not yet in the Designer.

```"github.com/devopsfaith/krakend-circuitbreaker/gobreaker": {
			"interval":        60,
			"timeout":         10,
			"name":            "fooo",
			"maxErrors":       5,
			"logStatusChange": true,
		},```

**Comment by alombarte** [comment]
Fixed in #84
